python/interpret_community/common/explanation_utils.py

```diff
@@ -43,13 +42,13 @@ def _summarize_data(X, k=10, to_round_values=True):
     :return: DenseData or SparseData object.
     :rtype: iml.datatypes.DenseData or iml.datatypes.SparseData
     """
-    is_sparse = sp.sparse.issparse(X)
+    is_sparse = issparse(X)
     if not isinstance(X, DenseData):
         if is_sparse:
             module_logger.debug('Creating sparse data summary as csr matrix')
             # calculate median of sparse background data
             median_dense = csc_median_axis_0(X.tocsc())
-            return sp.sparse.csr_matrix(median_dense)
+            return csr_matrix(median_dense)
         elif len(X) > 10 * k:
             module_logger.debug('Create dense data summary with k-means')
```

Comment:
[for future todo] Trying to get your thoughts here (it could be an overkill too) - but should we consider having file/files with all log messages so in future when we change log messages we don't have go through the entire code base. It might also avoid stale logs in case there are any?

Reply:
that's an interesting idea!  would that file contain string constants/variables only that would be referenced from other code, or specific functions that would be called for logging a particular message?  Trying to understand the mechanics for this idea.

Reply:
Mostly taking inspiration from my previous project for ACE team - we created a class which had functions return log messages which were reused across the project. 
https://github.com/Azure/AIPlatform/blob/master/end-to-end-solutions/Luna/src/Luna.Clients/Logging/LoggingUtils.cs
example - 
https://github.com/Azure/AIPlatform/blob/f0a6a2709ca7a6b2063d7237fe211c82ba1dd095/end-to-end-solutions/Luna/src/Luna.Services/Provisoning/ProvisioningService.cs#L117

but we also ended up having unique log strings in the file :P 
example - 
https://github.com/Azure/AIPlatform/blob/f0a6a2709ca7a6b2063d7237fe211c82ba1dd095/end-to-end-solutions/Luna/src/Luna.Services/Provisoning/ProvisioningService.cs#L184

And so I was thinking if it makes sense to use a similar approach where we maintain a common place for logging.

Reply:
There's also a question of how useful the logs are in the first place - I've only ever used them maybe twice, but maybe other people use them more often for debugging

Reply:
That is so true.